react/src/views/BoxEdit/BoxEditView.test.tsx

```diff
@@ -31,7 +33,31 @@ const initialQuery = {
   },
 };
 
-const succesfulMutation = {
+const initialQueryNetworkError = {
```

Comment:
Not sure if sensible but maybe in the future these error and query definitions can be moved into a separate file, too